src/main/java/com/mapr/synth/Synth.java

```diff
@@ -360,9 +433,12 @@ public ThreadReport() {
         @Option(name = "-count", handler = SizeParser.class)
         int count = 1000;
 
-        @Option(name = "-schema")
+        @Option(name = "-schema", required = false)
```

Comment:
The default is already false. This bit is garbage of experiments – should I make it required ?